resources/assetstore/charts/asset-metadata-service/templates/metrics.yaml

```diff
@@ -0,0 +1,22 @@
+# Inform Prometheus to scrap the metrics endpoint
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+    name: {{ template "fullname" . }}
+    labels:
+        prometheus: monitoring
+        app: {{ template "name" . }}
+        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+        release: "{{ .Release.Name }}"
+        heritage: "{{ .Release.Service }}"
+spec:
+    endpoints:
+        - interval: 10s
```

Comment:
To be consistent with other Service Monitors in Kyma, change it to 30s. Also, please update it for Asset Store, hCMS and Minio. You can even expose it in values 😄